lambdas/process-async-cri-credential/src/test/java/uk/gov/di/ipv/core/processasynccricredential/pact/JwtTestHelper.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.di.ipv.core.processasynccricredential.pact;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.nimbusds.jose.util.Base64URL;
+
+public class JwtTestHelper {
+    private final String minifiedHeaderJson;
```

Comment:
Minor thing, but I just noticed that these could be base 64 encoded in the constructor too.